text/0000-tiered-browser-support.md

```diff
@@ -0,0 +1,289 @@
+- [mdBook][] output (used for the books and other documentation)
+
+[urlo]: https://users.rust-lang.org/
+[irlo]: https://internals.rust-lang.org/
+[mdBook]: https://github.com/azerupi/mdBook/
+
+## Proposed browser support tiers
+
+Based on [actual usage metrics][] and with a goal of spending our time in an
+effective way, the browser support tiers would be defined as:
+
+[actual usage metrics]: #google-analytics-browser-usage-stats
+
+### Tier 1
+
+Goal: Ensure functionality of our web content for 80% of users.
+
+Browsers:
+
+- Chrome current stable and one previous release
+- Firefox current stable and the latest ESR
+- Safari current stable, one previous minor release, and the current version on
+  iOS
+- UC Browser current stable
+
+Support:
+
+- We add automated testing for each of these as much as possible (and work on
+  adding automated testing to those web contents that aren't currently tested,
+  such as rustdoc output).
+- Bugs affecting the functionality of the sites in these browsers are
+  prioritized highly.
+
+### Tier 2
+
+Goal: Have some support of our web content, without additional time, effort, or
+monetary cost, for about 90% of users.
+
+Browsers:
+
+- Chrome up to 3 releases back and 1 release ahead of the current stable
+- Firefox up to 2 releases back and 2 releases ahead of the current stable, one
+  previous ESR
+- Safari up to 2 releases back from the current stable
+- UC Browser 1 previous stable release
+- Edge current stable and 2 previous releases
+- IE 11 until its EOL
+- Android Webview current stable and 1 previous release
+- Opera latest stable
+
+Support:
+
+- No automated testing for these.
+- Bug reports for these browsers are left open but given a low priority, and
+  closed if they are still open when a browser is old enough to drop out of
+  this tier.
+- Pull requests to fix functionality for these browsers are accepted.
+
+### Tier 9999
+
+Otherwise known as "deliberately not supporting". Not tier 3, to avoid
+conflating with the Rustc tier 3 that still has some support.
+
+Goal: Avoid spending large amounts of time and code complexity debugging and
+hacking around quirks in older or more obscure browsers.
+
+Browsers:
+
+- Any not mentioned above
+
+Support:
+
+- No automated testing for these.
+- Bug reports for these browsers are closed as WONTFIX.
+- Pull requests to fix functionality for these browsers are accepted only if
+  they're deemed to not add an inordinate amount of complexity or maintenance
+  burden (subjective, reviewers' judgment).
+
+## Orthogonal but related non-proposals
+
+The following principles are assumptions I'm making that we currently follow
+and that we should continue to strive for, no matter what browser support
+policy we end up with:
+
+- Follow best practices for accessibilty, fix bug reports from blind users,
+  reach out to blind users in the community about how the accessibility of the
+  web content could be improved.
```

Comment:
+ 100 - I think making screen reader support (and other accessibility issues) a tier 1 (or possibly tier 1.5) issue is more important than supporting any particular browser.